extensions/amp-experiment/1.0/variant.js

```diff
@@ -191,3 +200,30 @@ function assertName(name) {
   userAssert(nameValidator.test(name),
       'Invalid name: %s. Allowed chars are [a-zA-Z0-9-_].', name);
 }
+
+/**
+ * Validates the variant schema of a config.
+ * @param {!JsonObject} variant
+ * @param {string} experimentName
+ * @param {string} variantName
+ * @throws {!Error}
+ */
+function assertVariant(variant, experimentName, variantName) {
+
+  // Assert that the variant is an object
+  userAssert(
+      isObject(variant),
+      `${experimentName}.${variantName} must be an object.`);
+
+  // Assert the variant weight
+  userAssert(
+      variant['weight'] !== undefined &&
+    typeof variant['weight'] === 'number',
+      `${experimentName}.${variantName} must have a weight.`);
+
+  // Assert the variant mutations
+  userAssert(
+      isArray(variant['mutations']),
```

Comment:
what will be included in the `mutations`? Is it still required if it is an empty array?

Reply:
check the length of the array as well